codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessageBuilders.java

```diff
@@ -604,14 +604,14 @@ public MqttSubAckMessage build() {
 
     public static final class UnsubAckBuilder {
 
-        private short packetId;
+        private int packetId;
         private MqttProperties properties;
         private final List<Short> reasonCodes = new ArrayList<Short>();
 
         UnsubAckBuilder() {
         }
 
-        public UnsubAckBuilder packetId(short packetId) {
+        public UnsubAckBuilder packetId(int packetId) {
```

Comment:
I think we need to keep the old method as well as otherwise this would be a breaking change